packages/es-components/src/components/containers/modal/Modal.js

```diff
@@ -227,6 +229,18 @@ function Modal({
     }
   }, [modalRef.current]);
 
+  useEffect(() => {
+    if (rootNode === document.body) return;
+
+    const targetNode = rootNode.querySelector(':host > *');
```

Comment:
Here we're assuming that the first element is the app element. This may not always be correct. (Sometimes it could be a style tag.) I'll fix that.